class02/exercise_5.py

```diff
@@ -35,7 +46,15 @@
 
 # code up your solution here
 
-
+for i in range(0,50):
```

Comment:
your answer is not printing out numbers  that are not Fizz or Buzz, check example output.